lib/src/message/message_item_bloc.dart

```diff
@@ -228,6 +234,12 @@ class MessageItemBloc extends Bloc<MessageItemEvent, MessageItemState> {
           unregisterListeners();
         }
       }
+    }else if (event.hasType(Event.msgFailed)){
```

Comment:
Shouldn't we also check for `_messageId == eventMessageId` here?

Reply:
Good point.